.eslintrc

```diff
@@ -29,6 +29,7 @@
     "react/jsx-handler-names": 0,
     "react/jsx-fragments": 0,
     "react/no-unused-prop-types": 0,
-    "import/export": 0
+    "import/export": 0,
+    "no-use-before-define": 0
```

Comment:
was getting errors about `React` not being defined, but that's not needed since the compiler inserts those already by itself depending on the `jsx` mode.

Reply:
checked that, but it doesn't work and crashes out, the rule may have been deleted

it's also documented as discouraged to use: https://palantir.github.io/tslint/rules/no-use-before-declare/

It applies pretty much only to `var` keywords, which we typically don't use, so I'll leave this as is.